yarn-project/end-to-end/src/e2e_pending_commitments_contract.test.ts

```diff
@@ -23,7 +23,7 @@ describe('e2e_pending_commitments_contract', () => {
     const blockNum = await aztecNode!.getBlockNumber();
     const block = (await aztecNode!.getBlocks(blockNum, 1))[0];
 
-    const commitmentsArray = block.body.txEffects.flatMap(txEffect => txEffect.newNoteHashes);
+    const commitmentsArray = block.body.txEffects.flatMap(txEffect => txEffect.noteHashes);
```

Comment:
Note hashes array while we are at it?

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4876/commits/4de7e87b319e213dd667cfe2af961b40cda7b903